### ISSUE TITLE
Get Alloy in the Mac App Store

### ISSUE THREAD

**Issue by pkriens** [AlloyTools/org.alloytools.alloy#9]

**Comment by zsfre** [comment]
I'm struggling to even get started on OS/X because of the enforced requirement for Java 6 (no longer supported from 10.14) when starting the app.

**Comment by zsfre** [comment]
And now I've discovered the 5.1.0 release at this website. Could someone update the main website? Thx

**Comment by pkriens** [comment]
> Probably it's better to put it to a [Homebrew](https://github.com/Homebrew/brew) instead. What do you think, @pkriens Peter?
@ligurio yes that would be nice ... but that would require some work I've not been able to find time for. PR's appreciated!

**Comment by pkriens** [comment]
Should be there soon ...

**Comment by pkriens** [comment]
https://github.com/Homebrew/homebrew-core/pull/210522
